### DISCUSSION TITLE
Mingda Magician printers added (X, Max, Pro)

### DISCUSSION THREAD

**Discussion by Matrekus** [OrcaSlicer/OrcaSlicer#131] [Ideas]
Hello, I worked out the resources for the Mingda Magicans. I started with the PRO, cause I have it myself and it works very well.
Is there an interest to join this printers?

Regards M.

![Softfever_MMPRO](https://user-images.githubusercontent.com/100630918/211162393-44f526b4-be71-4f90-b579-b3249a536f8e.png)

**Comment by Johana1000** [comment]
> Dobrý deň, vypracoval som zdroje pre Mingda Magicans. Začal som s PRO, pretože ho mám sám a funguje veľmi dobre. Je záujem pridať sa k týmto tlačiarňam?
> 
> S pozdravom M.
> 
> ![Softfever_MMPRO](https://user-images.githubusercontent.com/100630918/211162393-44f526b4-be71-4f90-b579-b3249a536f8e.png)

Can this be downloaded?
Thanks

**Reply by Johana1000** [reply]
Can this be downloaded?
Thanks
